### ISSUE TITLE
Kitchen 2.x: Could not load the 'ansible_playbook' provisioner from the load path.

### ISSUE THREAD

**Issue by getjack** [neillturner/kitchen-ansible#305]
Just updated to test-kitchen 2.0.1 randomly and tried to run kitchen test with kitchen-ansible.

**Comment by Normo** [comment]
I came across the same problem. I'm using test-kitchen 2.1.0 and kitchen-ansible 0.49.1.
```
-----> Starting Kitchen (v2.1.0)
>>>>>> ------Exception-------
>>>>>> Class: Kitchen::ClientError
>>>>>> Message: Could not load the 'ansible_playbook' provisioner from the load path. Did you mean: chef_apply, chef_base, chef_solo, chef_zero, dummy, shell ? Please ensure that your provisioner is installed as a gem or included in your Gemfile if using Bundler.
>>>>>> ----------------------
>>>>>> Please see .kitchen/logs/kitchen.log for more details
>>>>>> Also try running `kitchen diagnose --all` for configuration
```

**Comment by nqb** [comment]
Could you provide some details like `.kitchen/logs/kitchen.log` or `kitchen diagnose --all` ?

**Comment by Normo** [comment]
@nqb Here is the log file: [.kitchen/logs/kitchen.log](https://github.com/neillturner/kitchen-ansible/files/3107929/kitchen.log)

**Comment by Normo** [comment]
Probably the same problem as in #302

**Comment by neillturner** [comment]
fixed in version 0.50.0
